aas_core_meta/marker.py

```diff
@@ -62,18 +62,27 @@ class reference_in_the_book:
 
     @require(lambda section: all(number >= 1 for number in section))
     @require(lambda index: index >= 0)
-    def __init__(self, section: Tuple[int, ...], index: int = 0) -> None:
+    def __init__(
+        self, section: Tuple[int, ...], index: int = 0, fragment: str = ""
+    ) -> None:
         """
         Initialize with the given values.
 
         :param section: Section of the book given as tuple (so that it is sortable)
         :param index:
             Index in the section.
-
             The index helps us distinguish between multiple definitions in a section.
+        :param fragment:
+            Title of (Sub-)Section for anchor.
         """
         self.section = section
         self.index = index
+        if fragment == "":
+            classname = " "
+            fragment = ".".join(map(str, section)) + " " + classname + " Attributes"
+            fragment = fragment.replace(" ", "%20")
+
```

Comment:
```suggestion
```

The implementation has no real effect, but is, IMHO, a bit confusing to read (the class name is initialized as an empty string, but what I understand that you meant here is actually a placeholder). I translated it into text in the docstring.

Reply:
Is it possible to get the class that uses the marker or should we set the classname later?